components/sendgrid/actions/send-email-single-recipient/send-email-single-recipient.mjs

```
@@ -211,6 +212,16 @@ export default {
         },
       };
       attachments = this.getArrayObject(this.attachments);
+      attachments.map((attachment) => {
+        if (attachment.filepath) {
+          const filepath = this.checkTmp(attachment.filepath);
+          attachment.content = fs.readFileSync(filepath, {
+            encoding: "base64",
+          });
+        }
+        delete attachment.filepath;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)